src/state_machine.zig

```diff
@@ -231,6 +243,7 @@ pub fn StateMachineType(
                 .ids = constants.tree_ids.transfers_pending,
                 .batch_value_count_max = batch_value_count_max.transfers_pending,
                 .ignored = &[_][]const u8{"padding"},
+                .optional = &[_][]const u8{"status"},
```

Comment:
Zero means `TransferPendingStatus.none` (not pending), so it's not expected to have any object with `status == 0`.

I don't have a strong opinion here, the index could be considered either optional or required to be non-zero.

Reply:
Ah, I didn't realize that. I guess it doesn't matter in either case then.